src/commands/subcommands/labels.ts

```diff
@@ -0,0 +1,81 @@
+import { HttpStatusCode, IModify } from '@rocket.chat/apps-engine/definition/accessors';
+import { SlashCommandContext } from '@rocket.chat/apps-engine/definition/slashcommands';
+import { LayoutBlock } from '@rocket.chat/ui-kit';
+
+import { TodoistApp } from '../../../TodoistApp';
+import { MiscEnum } from '../../enums/Misc';
+import {
+  getActionsBlock,
+  getButton,
+  getContextBlock,
+  getSectionBlock,
+} from '../../helpers/blockBuilder';
+import { getLabelsUrl } from '../../helpers/const';
+import { ILabel } from '../../interfaces/labels';
+
+export async function labels(
+  app: TodoistApp,
+  modify: IModify,
+  context: SlashCommandContext
+): Promise<void> {
+  const user = context.getSender();
+  const room = context.getRoom();
+
+  const response = await app.getHttpHelperInstance().get(user, getLabelsUrl());
+
+  if (response.statusCode !== HttpStatusCode.OK) {
+    const msg = modify
+      .getCreator()
+      .startMessage()
+      .setText(`❗️ Unable to retrieve labels! \n Error ${JSON.stringify(response)}`)
```

Comment:
This often leaks stuff to the user, avoid returning network responses, those belong to logs.
Besides that, the message is good :)